bats/fb-katello-client.bats

```diff
@@ -12,6 +12,11 @@ load fixtures/content
   if [[ $KATELLO_VERSION != 4.[1-9]* ]]; then
     skip "Enabling katello-agent explicitly is only available with Katello 4.1+"
   fi
+
+  if tPackageExists qpid-cpp-server ; then
```

Comment:
I am trying to think if this could cause problems as this alone is not a true indicator if katello-agent is enabled.

Reply:
Yeah, it's quick and dirty. I could check the output of the ping api or something like that too, but I'd rather do so only if it's worth it

Reply:
Yea, I think we should drop this for now or derive it from `katello.yaml` or the answers file.

Reply:
dropping for now!